autoinstall/RHEL/9

```diff
@@ -0,0 +1 @@
+8
```

Comment:
I think this symbol link is not necessary, if we add symbol links for all the supported versions, then it will be disorderly. We can add a doc for guiding users that which autoinstall file works for which OS and versions.